python/hail/tests/test_api.py

```diff
@@ -611,7 +604,7 @@ def test_naive_coalesce(self):
         self.assertTrue(vds._same(repart))
 
     def tests_unions(self):
```

Comment:
does `tests_` also get tested, or does it need to be `test_unions`?

Reply:
I'll check!

Reply:
It did get tested. I've renamed it, though.